src/App.tsx

```diff
@@ -1,8 +1,58 @@
 import React from 'react';
+import { GoodsList } from './components/GoodsList';
+import { Button } from './components/Button';
 import './App.css';
 
-const App = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+const URL
+  = 'https://mate-academy.github.io/react_dynamic-list-of-goods/goods.json';
 
-export default App;
+interface AppState {
+  goods: Good[];
+}
+
+async function getGoods(url: string): Promise<Good[]> {
```

Comment:
if it is getGoods function - it shouldn't accept URL. If it accepts URL it should be named getFromApi | loadData etc.